docs/datamodel/annotations.rst

```diff
@@ -33,45 +32,30 @@ For example, consider the following declaration:
         }
     }
 
-The above annotations can be extracted via schema introspection queries
-and used to create a descriptive UI for an admin tool:
-
-.. code-block:: edgeql-repl
-
-    db> WITH MODULE schema
-    ... SELECT ObjectType {
-    ...     name,
-    ...     annotations: {
-    ...         name,
-    ...         @value
-    ...     }
-    ... }
-    ... FILTER .name = 'default::Status';
-    {
-        Object {
-            name: 'default::Status',
-            annotations: {
-                Object {
-                    name: 'std::description',
-                    @value: 'All possible user activities'
-                },
-                Object {
-                    name: 'std::title',
-                    @value: 'Activity status'
-                }
-            }
-        }
-    }
-
 The ``deprecated`` annotation is used to mark deprecated items (e.g.
 :eql:func:`str_rpad`) and to provide some information such as what
 should be used instead.
 
 
-See Also
---------
+User-defined annotations
+------------------------
+
+To declare a custom constraint type beyond the three built-ins, add an abstract
+annotation type to your schema.
```

Comment:
Perhaps mention that "For example, user-defined annotations can be used to attach arbitrary JSON metadata to the schema items."? I think it's cool and can spark some ideas for the reader